website/docs/r/vapp_nat_rules.html.markdown

```diff
@@ -0,0 +1,163 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vapp_nat_rules"
+sidebar_current: "docs-vcd-resource-vapp-nat-rules"
+description: |-
+  Provides a vCloud Director vApp NAT resource. This can be used to create, modify, and delete NAT rules.
+---
+
+# vcd\_vapp\_nat\_rules
+
+Provides a vCloud Director vApp NAT resource. This can be used to create,
+modify, and delete NAT rules in a [vApp network](/docs/providers/vcd/r/vapp_network.html).
+
+!> **Warning:** Using this resource overrides any existing NAT rules on vApp network. It's recommended to have only one resource per vApp and vApp network. 
```

Comment:
> one resource per vApp and vApp network

Shouldn't this be said in the vApp FW rules and vApp static routing resource too?

Reply:
added

Reply:
Don't forget the static routing page (it doesn't have the change yet). Careful though - in that PR I have already requested another change on that line.

Reply:
I won't touch static routing PR until this PR merged to it.